src/tests/ftest/util/AgentUtils.py

```diff
@@ -111,11 +164,28 @@ def stop_agent(client_list):
 
     client_list -- kill daos_agent on these hosts
     """
-
     if len(sessions) == 0:
         return
 
     # this kills the agent
     for client in client_list:
         cmd = "pkill daos_agent"
-        resp = subprocess.call(["ssh", client, cmd])
\ No newline at end of file
+        resp = subprocess.call(["ssh", client, cmd])
+
+    # check to make sure it's dead
+    time.sleep(5)
+    found_hosts = []
+    for host in client_list:
+        proc = subprocess.Popen(["ssh", host,
+                                 "pgrep 'daos_agent'"],
+                                stdout=subprocess.PIPE)
+        stdout = proc.communicate()[0]
+        resp = proc.wait()
+        if resp == 0:
+            found_hosts.append(host)
+
+    if found_hosts:
+        raise AgentFailed("daos agent processes {} found on hosts "
+                          "{} after stop_agent() were "
+                          "killed".format(', '.join(stdout.splitlines()),
```

Comment:
They weren't actually killed after they were found though.  It was the killing of them that failed if they were still found.

Reply:
The grammar on this is wonky, artifact of copying the message for the equivalent situation in ServerUtils. The intent is "hey user, we tried to pkill a process, it didn't work, here's a minor amount of debug information regarding this". Will change wording to reflect this.